front/tests/pages/stdcm-page-model.ts

```diff
@@ -260,7 +291,7 @@ class STDCMPage {
       this.destinationChField,
     ];
     for (const field of emptyFields) await expect(field).toHaveValue('');
-    await expect(this.codeCompoField).toHaveValue('__PLACEHOLDER__');
+    await expect(this.speedLimitTagField).toHaveValue('__PLACEHOLDER__');
```

Comment:
you may have conflicts with [this pr](https://github.com/OpenRailAssociation/osrd/pull/9987)